src/site/stages/build/drupal/graphql/eventPage.graphql.js

```diff
@@ -60,6 +64,10 @@ module.exports = `
       }
     }
     fieldEventRegistrationrequired
-    fieldAdditionalInformationAbo
+    ${
+      enabledFeatureFlags[featureFlags.FEATURE_FIELD_ADDITONAL_INFO]
+        ? 'fieldAdditionalInformationAbo'
+        : 'fieldAdditionalInformationAbo {processed}'
```

Comment:
I think this is reversed? So far we've been associating the query change that will be in place going forward with the feature flag being true.